patches/components-content_settings-core-browser-host_content_settings_map.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/content_settings/core/browser/host_content_settings_map.h b/components/content_settings/core/browser/host_content_settings_map.h
+index 68dd1557dc70ea36d531136f359e10d619c10750..50ba9d037c3e8adf7ee4e0c506a764bf5cb1b821 100644
+--- a/components/content_settings/core/browser/host_content_settings_map.h
++++ b/components/content_settings/core/browser/host_content_settings_map.h
+@@ -309,6 +309,7 @@ class HostContentSettingsMap : public content_settings::Observer,
+     return pref_provider_;
+   }
+ 
++  BRAVE_HOST_CONTENT_SETTINGS_MAP_H
```

Comment:
Please correct me if I am wrong, but it seems to me that current impl we have in `BraveHostContentSettingsMap` don't actually need to subclass `HostContentSettingsMap` since we're not actually needed to override any methods in it or accessing its private members expect for its destructor?
Things we're currently using in the implementation seems to be well accessible through `HostContentSettingsMap`'s public methods. 
Could we not inherit that and have our own class to achieve the same goal without patching  `host_content_settings_map_factory.cc` and `host_content_settings_map_factory.h`?

Reply:
I agree that there is no reason to subclass this

Reply:
Done 👍